src/commands/deployment/correlate-image.ts

```diff
@@ -0,0 +1,128 @@
+import {isAxiosError} from 'axios'
+import chalk from 'chalk'
+import {Command, Option} from 'clipanion'
+
+import {FIPS_IGNORE_ERROR_ENV_VAR, FIPS_ENV_VAR} from '../../constants'
+import {toBoolean} from '../../helpers/env'
+import {enableFips} from '../../helpers/fips'
+import {Logger, LogLevel} from '../../helpers/logger'
+import {retryRequest} from '../../helpers/retry'
+import {getApiHostForSite, getRequestBuilder} from '../../helpers/utils'
+
+export class DeploymentCorrelateImageCommand extends Command {
+  public static paths = [['deployment', 'correlate-image']]
+
+  public static usage = Command.Usage({
+    category: 'CI Visibility',
+    description: 'Correlate images with their source commit.',
+    details: `
+      This command will correlate the image with a commit of the application repository.
+    `,
+  })
+
+  private commitSha = Option.String('--commit-sha')
+  private repositoryUrl = Option.String('--repository-url')
+  private image = Option.String('--image')
+  private fips = Option.Boolean('--fips', false)
+  private fipsIgnoreError = Option.Boolean('--fips-ignore-error', false)
+  private dryRun = Option.Boolean('--dry-run', false)
+
+  private logger: Logger = new Logger((s: string) => this.context.stdout.write(s), LogLevel.INFO)
+
+  private config = {
+    apiKey: process.env.DATADOG_API_KEY || process.env.DD_API_KEY,
+    appKey: process.env.DATADOG_APP_KEY || process.env.DD_APP_KEY,
```

Comment:
could we get rid of `DATADOG_` env vars altogether? Since this is a new command we can use the chance not to carry deprecated naming